packages/react-examples/src/react/PeoplePicker/PeoplePicker.List.Example.tsx

```diff
@@ -20,6 +25,20 @@ const checkboxStyles = {
   },
 };
 
+const personaStyles = {
```

Comment:
Can you add a type here, probably `Partial<IPersonaStyles>`? This makes it more self-documenting and gives better intellisense and probably better error messages if something changes in the future.